components/portfolio/MarketPositions.tsx

```diff
@@ -125,197 +125,98 @@ export const MarketPositions = ({
     (pos) => pos.outcome == "Pool Share",
   );
 
-  return (
-    <div className={`${className}`}>
-      <MarketPositionHeader
-        marketId={market.marketId}
-        question={market.question}
-      />
-      <Table
-        columns={isLiquidityMarket ? COLUMNS_LIQUIDITY : COLUMNS}
-        data={positions.map<TableData>(
-          ({
-            assetId,
-            price,
-            userBalance,
-            outcome,
-            changePercentage,
-            market,
-            avgCost,
-            rpnl,
-            upnl,
-          }) => {
-            const baseAssetUsdPrice = lookUpAssetPrice(
-              market.baseAsset,
-              foreignAssetPrices,
-              usdZtgPrice,
-            );
+  //display balance if any of the positions is greater than 0.01 USD
+  const displayBalance = positions.some((pos) => {
+    const baseAssetUsdPrice = lookUpAssetPrice(
+      market.baseAsset,
+      foreignAssetPrices,
+      usdZtgPrice,
+    );
+    return (
+      pos.userBalance
+        .mul(pos.price)
+        .mul(baseAssetUsdPrice)
+        .div(ZTG)
+        .toNumber() >= 0.01
```

Comment:
Could you pull 0.01 out to an appropriately named variable so that it's clearer?